tests/monotouch-test/ModelIO/MDLLight.cs

```diff
@@ -41,6 +43,10 @@ public class MDLLightTest {
 		[TestFixtureSetUp]
 		public void Setup ()
 		{
+#if MONOMAC
+			if (!TestRuntime.CheckMacSystemVersion (10, 11))
```

Comment:
Same: replace `CheckSystemVersion` with call to `TestRuntime.CheckXcodeVersion`.